packages/teams-js/src/public/app.ts

```diff
@@ -761,15 +635,15 @@ export namespace app {
    */
   export function notifyAppLoaded(): void {
     ensureInitializeCalled();
-    sendMessageToParent(Messages.AppLoaded, [version]);
+    sendMessageToParentWithVersion(ApiVersion.V_2, Messages.AppLoaded, [version]);
   }
 
   /**
    * Notifies the frame that app initialization is successful and is ready for user interaction.
    */
   export function notifySuccess(): void {
```

Comment:
Since you are doing the mirror API's (old vs new) shouldn't this PR include the V1 version of these handlers in appInitializationts file?

Reply:
Yes, APIs will be gone through file by file. But for this batch, let's just focus APIs under app file and some of them under publicAPIs file (because some changes in app will affect them)